test/Dialect/FIRRTL/annotations-errors.fir

```diff
@@ -148,18 +138,18 @@ circuit Foo: %[[{"a":null,"target":"~Foo|Bar"}]]
 
 ; A target pointing at a non-existent component should error.
 
-; expected-error @+1 {{unapplied annotations with target '~Foo|Foo>x' and payload '[{a}]'}}
-circuit Foo: %[[{"a":null,"target":"~Foo|Foo>x"}]]
-  module Foo:
+; expected-error @+1 {{unapplied annotations with target '~FooNec|FooNec>x' and payload '[{a}]'}}
+circuit FooNec: %[[{"a":null,"target":"~FooNec|FooNec>x"}]]
+  module FooNec:
```

Comment:
This looks like a no-op change? Can you back this out? Or: is this an intentional change to show the difference between this and the next test?